packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "8.2.0",
+  "version": "8.3.0",
```

Comment:
I don't think we need to publish a new version when we're just updating storybook.

Reply:
Updated and changed label to ignore-for-release